src/containers/categories-menu/index.tsx

```diff
@@ -0,0 +1,84 @@
+import { useCallback, type MouseEvent } from 'react';
+
+import cn from 'lib/classnames';
+
+import { activeCategoryAtom } from 'store/sidebar';
+import { customCategoryAtom } from 'store/sidebar';
+import { activeWidgetsAtom } from 'store/widgets';
+
+import { useRecoilState } from 'recoil';
+
+import CATEGORY_OPTIONS from 'containers/sidebar/constants';
+import widgets from 'containers/widgets/constants';
+
+import { Checkbox, CheckboxIndicator } from 'components/checkbox';
+import Icon from 'components/icon';
+
+import CHECK_SVG from 'svgs/ui/check.svg?sprite';
+
+const Category = () => {
+  const [categorySelected, setCategory] = useRecoilState(activeCategoryAtom);
+  const [customWidgetSelection] = useRecoilState(customCategoryAtom);
+  const [, setActiveWidgets] = useRecoilState(activeWidgetsAtom);
+
+  const handleClick = useCallback(
+    (event: MouseEvent<HTMLButtonElement>) => {
+      event.preventDefault();
+      setCategory(event.currentTarget.value);
+      const widgetsFiltered = widgets.filter((widget) =>
+        widget.categoryIds.includes(event.currentTarget.value)
```

Comment:
I cant reproduce the error again but somewhere i found:
![Screenshot 2023-12-27 at 15 28 46](https://github.com/Vizzuality/mangrove-atlas/assets/51995866/9e9d31b8-1d24-450d-a890-2626f9565509)